spec/javascript/packages/document-capture/components/acuant-capture-spec.jsx

```diff
@@ -271,8 +271,13 @@ describe('document-capture/components/acuant-capture', () => {
         </AnalyticsContext.Provider>,
       );
 
+      const start = async (...args) => {
+        const { onFailure } = args[0];
```

Comment:
I think we can use this syntax here
```suggestion
      const start = async ({ onFailure }) => {
```

Reply:
Works for me. 👍🏻